Dockerfile

```diff
@@ -1,26 +1,14 @@
-FROM python:3.6-slim
+FROM python:3-slim
 
-ENV HOST=0.0.0.0
-ENV PORT=8080
-
-RUN apt-get update -y && \
-    apt-get install git make curl -y
+RUN apt-get -y update && \
+    apt-get -y install git make curl
 
 RUN useradd fdp && \
     mkdir /home/fdp && \
     chown fdp:fdp /home/fdp
 
 COPY . /home/fdp
 
-RUN mkdir /home/fdp/data
-COPY samples/minimal.ttl /home/fdp/data/config.ttl
-
 WORKDIR /home/fdp
 
-RUN pip install .
-
-EXPOSE ${PORT}
-
-CMD fdp-run /home/fdp/data/config.ttl
-
-HEALTHCHECK --interval=5s CMD curl --silent --fail ${HOST}:${PORT} || exit 1
+RUN pip install .
```

Comment:
I think your dockerfile should include a `CMD` command to start the fdp. Perhaps `fdp-run 0.0.0.0 8080` to start with the in memory store by default.

Reply:
`docker-compose up` should be used when run FDP with docker :-)

Reply:
I think you could have the `CMD` in the docker file (in case for some reason I want to use the docker image without docker compose or with my own docker compose file). This does not discard that you can give the command on docker compose file.

Reply:
You're right! Added already.